src/task/links.clj

```diff
@@ -1,5 +1,16 @@
-(ns task.links)
+(ns task.links
+  (:require
+    [task.office :as office]
+    [task.rent-place :as rent-place]))
 
-(def base "https://renterswarehouse.com")
+(def ^:dynamic *base-url* "https://renterswarehouse.com")
```

Comment:
why this became dynamic?

Reply:
You are correct, it's not necessary. 
I have copied from an example in my studies and had the same doubt. I've thought could improve the legibility perhaps.